plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -255,6 +252,138 @@ p. With this definition we can use expressions like @t.a.x@, see the image below
 p=. Content assist in data structure traversal
 
 
+h2(#cdom_pointers). Pointers
+
+Pointers are a core feature of the C programming language. YAKINDU Statechart Tools' Deep C Integration is making C pointers available to you in your statecharts. In particular, you can
+* declare typed pointer variables,
+* assign a pointer to a pointer variable of the same type,
+* retrieve the pointer pointing to a given variable,
+* pass pointers as parameters to functions, and
```

Comment:
In this case we are calling a C function directly without any detour to an operation. So I think *function* is correct.